hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh

```
@@ -59,9 +59,17 @@ PID_FILE="$BIN/pid"
 
 . "$BIN"/util.sh
 
+PARENT_DIR="$(cd "$TOP"/../ && pwd)"
+SERVER_VERSION_DIR="${SERVER_VERSION_DIR:-$(find_hugegraph_server_dir "$PARENT_DIR")}"
+
 ensure_path_writable "$LOGS"
 ensure_path_writable "$PLUGINS"
 
+# preload rocksdb/toplingdb
+if [ -n "$SERVER_VERSION_DIR" ] && [ -e "$SERVER_VERSION_DIR/bin/preload-topling.sh" ]; then
+    source "$SERVER_VERSION_DIR/bin/preload-topling.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./bin/preload-topling.sh was not specified as input (see shellcheck -x).

(SC1091)